Cargo.toml

```diff
@@ -8,9 +8,21 @@ members = [
 shared-version = true
 tag-name = "v{{version}}"
 
-# generated by 'cargo dist init'
+# Config for 'cargo dist'
+[workspace.metadata.dist]
+# The preferred cargo-dist version to use in CI (Cargo.toml SemVer syntax)
+cargo-dist-version = "0.0.2"
+# The preferred Rust toolchain to use in CI (rustup toolchain syntax)
+rust-toolchain-version = "1.67.1"
+# The CI to use
+ci = ["github"]
+# The installers to generate for each app
+installers = ["github-shell", "github-powershell"]
+# Target platforms to build apps for (Rust target-triple syntax)
+targets = ["x86_64-unknown-linux-gnu", "x86_64-apple-darwin", "x86_64-pc-windows-msvc"]
+
+
+# The profile that 'cargo dist' will build with
 [profile.dist]
 inherits = "release"
-debug = true
-split-debuginfo = "packed"
```

Comment:
Is this intended to change in the new version?

Reply:
Yeah I'm backing off this stuff for now because there are weird unreliable interactions with people's existing profiles. Will be revisited later. :/